python/cudf/setup.py

```diff
@@ -46,12 +50,19 @@
             "../../cpp/build/include",
             "../../thirdparty/cub",
             "../../thirdparty/libcudacxx/include",
+            ARROW_HOME + "/include",
```

Comment:
Should use `os.path.join` instead of using raw string concatenation.

Reply:
yep